tests/test_mm_packing_config.py

```
@@ -0,0 +1,187 @@
+            remove_unused_columns=False,
+        )
+        out = validate_config(cfg)
+        assert out.remove_unused_columns is False
+
+    def test_mm_packing_remove_unused_true_raises(self):
+        cfg = _cfg(
+            processor_type="AutoProcessor",
+            sample_packing=True,
+            remove_unused_columns=True,
+        )
+        with pytest.raises(Exception, match="(?i)remove_unused_columns"):
```

Ruff (0.15.20)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)